job/scriptsGen/normalizeSteps.js

```diff
@@ -137,6 +133,41 @@ function _normalizeNewFormatSteps(steps, defaultRuntime, onSuccess,
   return clonedSteps;
 }
 
+function __generateRuntimeInfo(task, buildJobId, buildScriptsDir,
+  buildStatusDir) {
+  var defaultENVs = {
+    SHIPPABLE_NODE_ARCHITECTURE: global.config.shippableNodeArchitecture,
+    SHIPPABLE_NODE_OPERATING_SYSTEM: global.config.shippableNodeOperatingSystem
+  };
+  var taskEnvs = _.extend({}, defaultENVs);
+  _.extend(taskEnvs, {
+    TASK_NAME: task.name || util.format('task_%s', task.taskIndex),
+    TASK_IN_CONTAINER: task.runtime.container
+  });
+  task.taskScriptFileName = util.format('task_%s.sh', task.taskIndex);
+  if (task.runtime.container) {
+    var containerName =  util.format('reqExec.%s.%s', buildJobId,
+      task.taskIndex);
+    task.runtime.options.options = util.format('%s --name %s',
+      task.runtime.options.options, containerName);
+    task.bootScriptFileName = util.format('boot_%s.sh', task.taskIndex);
+    // sets container task envs
+    var taskContainerEnvs = {
+      TASK_CONTAINER_OPTIONS: task.runtime.options.options,
```

Comment:
- we need to move all these in env file
- the env file should have `TASK_CONTAINER_OPTIONS=<%=obj.taskContainerOptions%> and this file should just generate the `taskContainerOptions` object

Reply:
there should be no bash components here.